src/index.ts

```diff
@@ -69,8 +71,21 @@ async function run() {
 
   const labels = uniq(
     connectedLabelsResponses.reduce<string[]>((acc, response) => {
-      const issueLabels = response.data.map((label) => label.name)
-      return [...acc, ...issueLabels]
+      const issueLabels = response.data.map((label) => label.name);
+      // Filter out unwanted labels and keep only the ones that are needed
+      const filteredLabels = issueLabels.filter(label => {
+          if (labelsToCopy.length > 0 && !labelsToCopy.includes(label)) {
+              // Label not in labelsToCopy
```

Comment:
```suggestion
              // Label not in `labelsToCopy`
```

You can use markdown formatting in code comments. Formatting will also be applied in your text editor.